modules/security-context-constraints-about.adoc

```diff
@@ -143,10 +143,7 @@ The `restricted` SCC:
 * Allows pods to use any FSGroup
 * Allows pods to use any supplemental group
 
-[NOTE]
-====
-The restricted SCC is the most restrictive of the SCCs that ship by default with the system. However, you can create a custom SCC that is even more restrictive. For example, you can create an SCC that restricts `readOnlyRootFS` to `true` and `allowPrivilegeEscalation` to `false`.
```

Comment:
In SCC yaml, the field name is `readOnlyRootFilesystem`, rather than `readOnlyRootFS`. Replace readOnlyRootFS with readOnlyRootFilesystem.